public_html/wp-content/mu-plugins/blocks/package.json

```diff
@@ -52,6 +52,7 @@
 	"scripts": {
 		"start": "wp-scripts start blocks=./source/blocks.js live-schedule=./source/blocks/live-schedule/front-end.js live-posts=./source/hooks/latest-posts/front-end.js",
 		"build": "wp-scripts build blocks=./source/blocks.js live-schedule=./source/blocks/live-schedule/front-end.js live-posts=./source/hooks/latest-posts/front-end.js",
+		"lint:js:comment": "Usage: `npm run lint:js source/blocks/schedule -- --fix`. The path has to be from the root `mu-plugins/blocks` folder.",
```

Comment:
It's intended to be a comment, but `npm` will strip actual comments out. There are [a lot of workarounds for that](https://stackoverflow.com/questions/14221579/how-do-i-add-comments-to-package-json-for-npm-install), and this one seemed to have the best tradeoffs in this context.

I didn't intend people to run it, but I'll add an `echo` before it for anyone who does.

Reply:
I think it's necessary. I added it because I tried a lot of variations and couldn't get any to work. After digging through a lot of stuff I realized the problem was that the path had to be relative to the root folder, which didn't feel intuitive to me. I don't think the external documentation is close enough or detailed enough to save people from running into the same problems that I did.

Reply:
I personally prefer `npm run {command}`, because it's shorter and more memorable, and I rarely run a `blocks/` command from outside the `blocks/` folder.

If you both prefer the `yarn` variation, though, I'm happy to use that for the comment.

Reply:
> It's intended to be a comment, but npm will strip actual comments out.

Right, that's why this sort of documentation usually goes in a readme. I think this clutters the list of scripts, and is confusing since it's not a real command. I'm ➕  for adding documentation, but in the readme.

Reply:
I agree that it's super confusing to have a comment here. I'd prefer to have it in the readme.